release-team/role-handbooks/release-notes/editing-flow.md

```diff
@@ -0,0 +1,245 @@
+
+Would you like to continue from the last session? (y/n):
+``` 
+
+#### Reviewing and editing the release notes
+
+Once started, the flow will present the user each release note to
+review. If the user's notices a field that needs to be changed, 
+they might choose to edit that note. Here is an example:
+
+```
+Release Note for PR 92546:
+==========================
+Pull Request URL: https://github.com/kubernetes/kubernetes/pull/92546
+    Author: @liggitt
+    SIGs: [api-machinery]
+    Kinds: [cleanup]
+    Areas: [custom-resources]
+    Feature: false
+    ActionRequired: false
+    ReleaseVersion: 1.19.0
+    Text:
+    │ kube-apiserver: openapi schemas published for custom resources now reference
+    │ standard ListMeta schema definitions
+
+- Fix note for PR #92546? (y/N): 
+
+```
+
+If the user chooses to edit the note when prompted, krel will open the current note
+in the default editor (defined in `$EDITOR`) for the user to edit:
+
+```
+---
+# This is the current map for this Pull Request.
+# The original note content is commented out, if you need to
+# change a field, remove the comment and change the value.
+# To cancel, exit without changing anything or leave the file blank.
+# Important! pr: and releasenote: have to be uncommented.
+#
+# pr: 92546
+# releasenote:
+#   text: 'kube-apiserver: openapi schemas published for custom resources now reference
+#     standard ListMeta schema definitions'
+#   documentation: []
+#   author: liggitt
+#   areas:
+#   - custom-resources
+#   kinds:
+#   - cleanup
+#   sigs:
+#   - api-machinery
+#   feature: false
+#   action_required: false
+#   release_version: 1.19.0
+~
+~                                  
+~                       
+```
+
+By default, all the note's fields are commented out. The user can choose one or more
+fields to uncomment and modify. If at least one field is modified, a [release note map
+file](https://github.com/kubernetes/release/blob/master/docs/release-notes-maps.md) 
+will be created in the release directory after saving the file and exiting the editor.
+
+__Note:__ A valid map file should have at least the PR number and the releasenote struct.
+
+If the user makes an error (a YAML syntax error for example), krel will prompt the user
+to retry editing the note:
+
+```
+ERRO The yaml code does not have a PR number      
+- An error occurred while editing PR #92546. Try again? (y/n)
+```
+
+#### Exiting the review loop
+
+When in the interactive cycle, krel will present the user with every release note that
+has not yet been reviewed. It is not necessary to review all the notes in one go, at
+any time the user may hit Ctrl+C and the interactive flow will exit at that point.
+
+To continue the editing process at a later time, `krel` will save its state
+in the release directory. It will store information about each review session inside of
+the `release-notes/sessions/`. The JSON files include the user's data and the release
+notes reviewed during the session.
```

Comment:
```
To continue the editing process at a later time, `krel` will save its state
in the release directory. It will store information about each review session inside of
the `release-notes/sessions/`. The JSON files include the user's data and the release
notes reviewed during the session.
```

Question, @JamesLaverack (Possibly a lead, somewhere, too? Unsure who to tag, possibly Lauri or Jeremy):

Is the expectation that whomever is running the command for a particular RC reviewing *every single note* for that given release? Is there a set minimum you want shadows to review in one run of the tool?